src/Build/Definition/Project.cs

```diff
@@ -3052,6 +3052,7 @@ internal Data(Project project, PropertyDictionary<ProjectPropertyInstance> globa
             public ItemDictionary<ProjectItem> Items { get; private set; }
 
             public List<ProjectItemElement> EvaluatedItemElements { get; private set; }
+            public IReadOnlyCollection<string> ExportTargets { get; set; }
```

Comment:
This should probably have comments since it's public?

Reply:
It's in the Data nested class which is not visible.